static/styles/zulip.css

```diff
@@ -1909,6 +1909,21 @@ nav a .no-style {
     letter-spacing: normal;
 }
 
+#tab-indicator {
+    z-index: 2;
+    padding-top: 0px;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    float: right;
+    letter-spacing: normal;
+    height: 40px;
```

Comment:
This should be around `33px` so that it does not cross the `searchbox input`. 

![image](https://user-images.githubusercontent.com/2263909/36342085-11fda830-141f-11e8-8e44-b6d1e1a85075.png)